core/invoker/src/main/resources/application.conf

```diff
@@ -93,6 +93,7 @@ whisk {
       dns-servers: []
       dns-search: []
       dns-options: []
+      extra-env-vars: [] # sequence of `key` and/or `key=value` bindings to add to all user action container environments
```

Comment:
If we introduce a fresh thing anyway, why not make it a map from the getgo to alleviate any parsing issues?

Reply:
reasonable point.  I'm on vacation until Wed; will make the change when I get back.

Reply:
I prototyped using a map instead of a seq[k=v] and found an unfortunate interaction with `transformEnvironment.sh` that makes a map cumbersome to use.   The problem is `transformEnvirovnment.sh` rewrites `_` to `.` which makes it quite cumbersome to define environment variables whose name contains an `_` without adding an escape sequence.
For example, consider trying to set `__OW_ALLOW_CONCURRENT=false` via a `CONFIG_` envar.  With a Seq, it is:
```
CONFIG_whisk_containerFactory_containerArgs_extraArgs_env_0=__OW_ALLOW_CONCURRENT=false
```
With a Map, it would be 
```
CONFIG_whisk_containerFactory_containerArgs_extraArgs_env___OW_ALLOW_CONCURRENT=false
```
but we also need to somehow escape all the `_` after `env_` to prevent `transformEnvironment.sh` from converting them to `.`. 

So, I think using Seq to avoid needing to escape `_` is a better trade-off overall.